src/store/main.js

```diff
@@ -115,6 +115,19 @@ export default new Vuex.Store({
 				state.boards.push(board)
 			}
 		},
+
+		cloneBoard(state, board) {
```

Comment:
I'd say we can call this updateBoard, so we can reue that one later if needed.

Reply:
There is already a updateBoard Action Methode. 
Maybe it's confusing if I rename the mutation from clone to update?

Reply:
Fine then :+1: